src/sbd-inquisitor.c

```diff
@@ -401,7 +390,7 @@ int cluster_alive(bool all)
     }
 
     for (s = servants_leader; s; s = s->next) {
-        if (sbd_is_disk(s) == false) {
+        if (sbd_is_cluster(s) || sbd_is_pcmk(s)) {
```

Comment:
it would be more efficient to return 1 if sbd_is_disk() before the loop, and skip this test altogether